frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -1,27 +1,93 @@
-import React, { createContext, useContext, ReactNode } from 'react';
-import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import React, { createContext, useContext, ReactNode, useState } from 'react';
+import {
+  MutationCache,
+  MutationMeta,
+  QueryCache,
+  QueryClient,
+  QueryClientConfig,
+  QueryClientProvider,
+  QueryMeta,
+} from '@tanstack/react-query';
 import type * as queries from '../api/queries';
 import type * as mutations from '../api/mutations';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
+import { ToastContainer, Slide, toast } from 'react-toastify';
+import { ErrorBoundary } from 'react-error-boundary';
+import { AxiosError } from 'axios';
+import { Trans, useTranslation } from 'react-i18next';
+import { Link } from '@digdir/design-system-react';
+import { ErrorBoundaryFallback } from '../components/ErrorBoundaryFallback';
+
+import 'react-toastify/dist/ReactToastify.css';
+import 'app-shared/styles/toast.css';
 
 export type ServicesContextProps = typeof queries & typeof mutations;
 export type ServicesContextProviderProps = ServicesContextProps & {
   children?: ReactNode;
-  client?: QueryClient;
+  client?: QueryClient; // TODO : should probably be removed to force the use of QueryCache and MutationCache
```

Comment:
I agree that this should be removed. Suggest we create an issue about it, and then we can clean this up within a separate pull request. Could we add the issue to the todo comment?
```suggestion
  client?: QueryClient; // TODO : #issue-id should probably be removed to force the use of QueryCache and MutationCache
```

Reply:
Good idea! I created the issue (#10913) and added it to the todo comment 👍